pkg/msg/v1alpha1/msg.go

```diff
@@ -61,38 +66,44 @@ func (m *msg) GoString() string {
 // msgPredicate abstracts evaluation of a message condition
 type msgPredicate func(given *msg) bool
 
+// IsInfo returns true if given message's MType is InfoMsg
```

Comment:
How about changing all the comments to `xxx returns true if given message is of type xxx`

Reply:
Agreed.